tests/test_minimal/test_tools/test_importing.py

```diff
@@ -11,5 +11,21 @@ def test_guide_attributes():
     for name, summary in summaries.items():
         for key, value in summary.items():
             assert value is not None, f"{name} is missing GUIDE related attribute {key}."
-            if isinstance(value, tuple):
+            if key == "keywords":
+                assert isinstance(
+                    value, (list, tuple)
+                ), f"{name} incorrectly specified GUIDE-related attribute 'keywords' (must be list or tuple)."
```

Comment:
Do we have current cases where it is a list instead of a tuple? In general I would move to have it be immutable (and thus constructed by partial copy if you want to 'inherit' from a parent attribute)

Reply:
At least MedPC uses a list, not sure about others.

Reply:
we shall see from the tests